models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -189,10 +189,10 @@ private function doGetDataForEditMode($data, $object, &$fieldData, &$metaData, $
         $inheritanceAllowed = $class->getAllowInherit();
         $inherited = false;
 
-        $dataItems = $data->getInternalData(true);
+        $dataItems = $data->getInternalData($params['objectFromVersion'] ? false : true);
```

Comment:
Could be simplified with `!$params['objectFromVersion']`?
Why is there no check if the key exists as done below?